Yoast/wpseo-woocommerce
# Replace browserify with webpack and configure yoastseo as an external

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Replaced browserify with webpack.
* Imported YoastSEO as a external so it relies on the version provided by WordPress SEO (free).

## Test instructions

This PR can be tested by following these steps:

0. ensure you have the latest version of the trunk branch of [wordpress seo](https://github.com/Yoast/wordpress-seo) installed in your plugin directory.
1. Checkout this branch
2. Run `yarn install`
3. Run `grunt build`
4. Verify the js files get build.
5. Generate a zip file by running `grunt create:artifact`
6. Test the functionality of the js files, they should work and not give errors.

Fixes #264
